packages/arb-bridge-peripherals/scripts/deploy_token_bridge_l1.ts

```diff
@@ -32,7 +32,7 @@ const main = async () => {
   const l2SignerAddress = await l2Signer.getAddress()
 
   // parse needed vars
-  const inboxAddress = l1Network.tokenBridge.inbox
+  const inboxAddress = await bridge.l1Bridge.getInbox()
   if (!inboxAddress) throw new Error('Please set inbox address! INBOX_ADDRESS')
```

Comment:
I think this check no longer makes sense? Idk if we want to replace with a check for `bridge.l1Bridge` above the `getInbox` call

Reply:
ye we can just remove it entirely; `.l1Bridge` will always be present, and  `getInbox()` will itself throw when appropriate